### PR TITLE
Migrate to solidity 0.6

### PR DESCRIPTION
**Pull request by vanruch**

### CODE REVIEW

File: test/helpers/bytes32.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = function bytes32(str) {
+    return web3.utils.padRight(web3.utils.toHex(str), 64);
```
Comment (krzysztof-jelski) [inline test/helpers/bytes32.js:2]: Can we use `ethers.formatBytes32String` instead?

File: contracts/Proxy/OwnedUpgradeabilityProxy.sol [hunk 2]
```diff
@@ -59,7 +60,7 @@ contract OwnedUpgradeabilityProxy {
 
     /**
     * @dev Tells the address of the owner
-    * @return the address of the owner
+    * @return pendingOwner the address of the owner
```
Comment (krzysztof-jelski) [inline contracts/Proxy/OwnedUpgradeabilityProxy.sol]:
```suggestion
    * @return pendingOwner the address of the pending owner
```

File: test/helpers/bytes32.js (hunk 1 above)
Comment (vanruch) [reply test/helpers/bytes32.js:2]: this would require adding ethers and anyway those helpers will disappear after we merge repos

Comment (krzysztof-jelski) [reply test/helpers/bytes32.js:2]: 👍

**Review by krzysztof-jelski — APPROVED** [review]

**Pull request merged**